changes/1817.feature.rst

```diff
@@ -1 +1 @@
-TextInput now supports focus handlers and changing alignment on GTK.
```

Comment:
This is the content that should be in 1821.feature.rst. 1817.feature.rst shouldn't be modified at all.